collectfast/storage_extensions/gcloud.py

```diff
@@ -0,0 +1,26 @@
+import base64
+import binascii
+
+from google.api_core.exceptions import NotFound
+
+from collectfast.storage_extensions.base import BaseStorageExtensions
+
+
+class GoogleCloudStorageExtensions(BaseStorageExtensions):
+    """
+    Storage extensions for django-storage's `GoogleCloudStorage`
+    """
+
+    def get_etag(self, path):
+        normalized_path = path.replace('\\', '/')
+        try:
+            md5_base64 = self.storage.bucket.get_blob(normalized_path)._properties['md5Hash']
+            return '"' + binascii.hexlify(base64.urlsafe_b64decode(md5_base64)).decode("utf-8") + '"'
```

Comment:
Is there a specific reason you are using `binascii.hexlify()` and `base64.urlsafe_b64decode()`as opposed to just a call to e.g. `base64.b64decode(md5_base64).decode("utf-8")`? I think it would be worth it to explain this line with a comment :)